client/components/TestQueueRow/index.jsx

```diff
@@ -530,30 +510,24 @@ const TestQueueRow = ({
                     </div>
                 </td>
                 <td>
-                    <div className="status-wrapper">{status}</div>
+                    <div className="status-wrapper">{labelStatus}</div>
                     {isSignedIn && isTester && (
                         <div className="secondary-actions">
-                            {isAdmin && !isLoading && nextReportStatus && (
+                            {isAdmin && !isLoading && (
                                 <>
                                     <Button
                                         ref={updateTestPlanStatusButtonRef}
                                         variant="secondary"
                                         onClick={async () => {
                                             focusButtonRef.current =
                                                 updateTestPlanStatusButtonRef.current;
-                                            await updateReportStatus(
-                                                nextReportStatus
-                                            );
+                                            await updateReportStatus();
                                         }}
                                     >
-                                        Mark as{' '}
-                                        {capitalizeEachWord(nextReportStatus, {
-                                            splitChar: '_'
-                                        })}
+                                        Mark as Final
                                     </Button>
```

Comment:
I noticed when I clicked this button to "Mark as Final" that it would work even if there were no actual results collected. That could cause confusion, because on the Data Management page, when advancing a TestPlanVersion to Candidate or Recommended, it does check whether any results were collected and fails. I think there should be a check before changing the status to make sure results actually exist.

Reply:
Good catch and should have never worked like that. I've addressed this in https://github.com/w3c/aria-at-app/pull/715/commits/f904219eb943afff452f61138717d02c06184441